core/config/toml/types.go

```diff
@@ -2434,6 +2437,7 @@ type Telemetry struct {
 	ChipIngressInsecureConnection *bool
 	HeartbeatInterval             *commonconfig.Duration
 	LogStreamingEnabled           *bool
+	LogLevel                      *string
```

Comment:
We usually try to keep these alphabetical:
```suggestion
	LogLevel                      *string
	LogStreamingEnabled           *bool
```